apps/condo/domains/resident/tasks/helpers/notifyOnPaydayisCurrnetDateValid.js

```diff
@@ -0,0 +1,13 @@
+const dayjs = require('dayjs')
+
+function notifyOnPaydayIsCurrentDateValid (today) {
```

Comment:
Any date can be passed to this function, so, a "date" name will reflect this:

```suggestion
function notifyOnPaydayIsCurrentDateValid (date) {
```